data/ecosystems/s/skale-network.toml

```diff
@@ -1,17 +1,78 @@
 # Ecosystem Level Information
 title = "Skale Network"
 
-sub_ecosystems = ["Owl Protocol", "Razor Network"]
+sub_ecosystems = [
+    "3Air",
+    "Bit Hotel",
+    "BlockDaemon",
+    "Blockscout",
```

Comment:
General ecosystem

Reply:
Removed BlockDaemon and Blockscout: 198b290f